app/models/mapping.rb

```diff
@@ -11,13 +11,13 @@ class Mapping < ActiveRecord::Base
 
   SUPPORTED_TYPES = %w(redirect archive unresolved)
 
-  acts_as_taggable
+  acts_as_ordered_taggable
   has_paper_trail :ignore => [:tag_list => Proc.new { |mapping|
     # tag_list appears to ActiveModel::Dirty/paper_trail to be an
     # ActsAsTaggableOn::TagList but the original value from the database is a
     # string. This means that it is "different" even though it isn't.
     # Comparing the stringified version avoids that problem.
-    mapping.tag_list_was == mapping.tag_list.to_s
+    mapping.tag_list_was == mapping.tag_list
```

Comment:
You probably want to leave this as it was.  As the comment says, `mapping.tag_list_was` is a string, but `mapping.tag_list` is a `ActsAsTaggableOn::TagList` object so this comparison will always fail unless you stringify the tag_list.